mod/homepage/lib/javascript/script_homepage_carousel.js.coffee

```diff
@@ -122,10 +135,11 @@ activateIntroTab = (tab)->
   active_panel.find('.carousel').carousel()
   active_panel.find('.carousel-item').first().addClass 'active'
 
-getNumbers = () -> 
+getNumberElements = () -> 
   values = []
-  $('.text-right.mx-3').each -> 
-    values.push( $(this).find('h1.font-weight-normal') )
+  $('h1.font-weight-normal._count-ele').each -> 
```

Comment:
`h1.font-weight-normal` is not a good selector.  It means the js will break if the style changes.

For javascript elements we typically prefer to use selectors that start with underscore. (and those selectors should NOT be used for styling)

Reply:
`$('h1._count-ele')`
yes?

Reply:
Why not just `_count-ele`? Is `_count-ele` used anywhere that isn't supposed to be counted?

Reply:
ok